processor/src/main/java/org/mapstruct/ap/internal/util/Filters.java

```diff
@@ -36,11 +36,11 @@
     private Filters() {
     }
 
-    public static List<Accessor> getterMethodsIn(List<Accessor> elements) {
+    public static List<Accessor> getterMethodsIn(AccessorNamingUtils namingUtils, List<Accessor> elements) {
         List<Accessor> getterMethods = new LinkedList<Accessor>();
 
         for ( Accessor method : elements ) {
-            if ( Executables.isGetterMethod( method ) ) {
+            if ( namingUtils.isGetterMethod( method ) ) {
```

Comment:
readability use full name at these places.: accessorNamingUtil or just  accessorNaming (expressing the function better)

Reply:
`accessorNaming` it is then